presto-tests/src/main/resources/META-INF/services/org.testng.ITestNGListener

```diff
@@ -0,0 +1 @@
+io.prestosql.tests.LogTestExecutionListener
```

Comment:
does it work for presto-main?

Reply:
It would need to go in `presto-main` instead to work there